test/test_dns_instrument.py

```diff
@@ -1,6 +1,14 @@
+import os
+
+import pytest
+
 from openwpm.utilities import db_utils
 
 
+@pytest.mark.skipif(
+    "CI" not in os.environ or os.environ["CI"] == "false",
+    reason="Makes remote connections",
+)
 def test_name_resolution(default_params, task_manager_creator):
```

Comment:
We need to connect to a DNS server here, so unless we start one ourselves this is an external connection